mobile/ci/linux_ci_setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+# Download and set up ndk 21 after GitHub update
```

Comment:
if you use the mobile build image this shouldnt be necessary - i think it was why it was added

Reply:
my misgiving is that it looks like another install to me, when i think we should be going in the opposite direction - but i dont know the specifics

Reply:
This is for running the emulator. Running the emulator inside Docker, although not impossible, requires a lot of work, such as installing the GUI libraries, etc. We can explore that option in the future once we have a stable CI. For now the goal of this PR is simply to test switching to Linux with a large runner.